### PR TITLE
Skip pending regression tests

### PR DESCRIPTION
**Pull request by mossprescott**
This also makes skipping pending tests automatic for QuasarSpecification. We were previously using it inconsistently.

### CODE REVIEW

**Comment by mossprescott** [comment]
Hmm. One failure in DataServiceSpec and one "no output for 10 minutes" so far.

File: foundation/src/test/scala/quasar/QuasarSpecification.scala [hunk 1]
```diff
@@ -22,7 +22,7 @@ import org.specs2.mutable._
 import org.specs2.scalaz.ScalazMatchers
 import scalaz._
 
-trait QuasarSpecification extends SpecificationLike with ScalazMatchers {
+trait QuasarSpecification extends SpecificationLike with ScalazMatchers with PendingWithAccurateCoverage {
```
Comment (paulp) [inline foundation/src/test/scala/quasar/QuasarSpecification.scala]: I'd say if QuasarSpecification is going to extend PendingWithAccurateCoverage and everything is going to extend QuasarSpecification, then there's no benefit to PendingWithAccurateCoverage existing at all, and you could just move its contents into QuasarSpecification.

**Comment by paulp** [comment]
Other than the one quibble, looks great to me.

File: foundation/src/test/scala/quasar/QuasarSpecification.scala (hunk 1 above)
Comment (mossprescott) [inline foundation/src/test/scala/quasar/QuasarSpecification.scala]: I wouldn't make the assumption that _everything_ will _always_ extend QuasarSpec., but on the other hand it's not that hard to change from one to the other, so :shrug:.

**Pull request merged**